apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -117,6 +118,34 @@ txs_gc(Config) ->
 
     ok = aecore_suite_utils:check_for_logs([dev1], Config).
 
+
+micro_block_cycle(Config) ->
+    MBC = aec_governance:micro_block_cycle(),
+    aecore_suite_utils:start_node(dev1, Config),
+    N1 = aecore_suite_utils:node_name(dev1),
+    aecore_suite_utils:connect(N1),
+
+    %% Mine a block to get some funds. Height=1
+    aecore_suite_utils:mine_key_blocks(N1, 1),
+
+    StartRes = rpc:call(N1, aec_conductor, start_mining, [], 5000),
+    ct:log("aec_conductor:start_mining() (~p) -> ~p", [N1, StartRes]),
+
+    [ begin
+        add_spend_tx(N1, 1000, 1,  Nonce, 10000),
+        timer:sleep(MBC div 100)
```

Comment:
So you sleep for a 1/100th of the micro block sleep time, with 20 Txs, will that ever span over more than 1 key block? I would suggest 1/2 of the micro block sleep time or something?!